### ISSUE TITLE
IMMORTAL TREASURE 1 2022

### ISSUE THREAD

**Issue by MasterOfUniverse82** [ValveSoftware/Dota2-Gameplay#5807]
I'm at compendium level 110 and I've unlocked a new Immortal Treasure 1. I already have six, so I already have six unlocked items, logically the least rare should come out, in this case BristleBack.
But something doesn't work, I dropped an item that I already own, and the others reset, as if I didn't have any of them.
There is a problem to solve!

**Comment by LokeLok** [comment]
This is how it works, not a bug.

**Comment by MasterOfUniverse82** [comment]
No, it doesn't work like that, the same item you own doesn't come out. The ones you already own are blacked out to let you know that you already own them, and roulette is for the ones you don't own.
But here they all reset, and a duplicate came out.
There is another similar ticket opened by another user. lol

**Comment by LokeLok** [comment]
When you have gotten all the items once. It resets and starts again. Rare items has a chance to drop, you're not guaranteed to get the Bristle rare.

**Comment by TheUnholyWrath** [comment]
In the dropdown under "Each treasure contains 1 of these items": "Until it's purchased and opened, you won't know which one you'll receive. If you buy more than one, you won't receive duplicates until you own one of each non-rare item. 
**Also, if you're very lucky, in addition to the normal items you receive, you may also receive one of the rare bonus items.**"

You're never guaranteed to get any of the rare items, but there are escalating odds of opening them. Please close this issue.
